src/java/org/apache/cassandra/auth/jmx/AuthorizationProxy.java

```diff
@@ -490,5 +491,17 @@ protected JMXPermissionsCache()
                   AuthorizationProxy::loadPermissions,
                   () -> true);
         }
+
+        public void invalidatePermissions(String roleName)
+        {
+            invalidate(RoleResource.role(roleName));
+        }
+    }
+
+    public static interface JmxPermissionsCacheMBean extends AuthCacheMBean
+    {
+        public static final String CACHE_NAME = "JMXPermissionsCache";
```

Comment:
Even though the class is renamed, I've kept the same name for JMX bean. It does not affect anything. Please, let me know if it is safe to rename the bean name as well.

Reply:
We need to keep backward compatibility for the JMX names. Some applications might be relying on them.